doc/UserGuide.md

```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorts all persons lexiographically : `sort`
+Sorts all persons in the address book in lexiographic order.<br>
```

Comment:
It might be better not to use uncommonly used words such as "lexiographic" as users may find it harder to understand.